lib/streamlit/server/UploadFileRequestHandler.py

```diff
@@ -84,6 +89,7 @@ def _require_arg(args, name):
         # Convert bytes to string
         return arg[0].decode("utf-8")
 
+
```

Comment:
Sanity-checking if this whitespace addition was intentional.  PEP-8 recommends: 

> Method definitions inside a class are surrounded by a single blank line

https://www.python.org/dev/peps/pep-0008/#blank-lines